lib/SQLiteReporter.cpp

```diff
@@ -215,6 +206,24 @@ void Mutang::SQLiteReporter::reportResults(const std::unique_ptr<Result> &result
   outs() << "Results can be found at '" << databasePath << "'\n";
 }
 
+#pragma mark -
+
+std::string SQLiteReporter::getCallerPathAsString(std::vector<std::string> callerPath) {
```

Comment:
This is what goes beyond my limited knowledge of C++. Should const go like:

```
std::string getCallerPathAsString(std::vector<std::string> const &callerPath);
```

or 

```
std::string getCallerPathAsString(const std::vector<std::string> &callerPath);
```

?